azure-mgmt-search/src/main/java/com/microsoft/azure/management/search/implementation/AdminKeysImpl.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+package com.microsoft.azure.management.search.implementation;
+
+import com.microsoft.azure.management.resources.fluentcore.model.implementation.WrapperImpl;
+import com.microsoft.azure.management.search.AdminKeys;
+
+/**
+ * Response containing the primary and secondary admin API keys for a given Azure Search service.
+ */
+class AdminKeysImpl extends WrapperImpl<AdminKeyResultInner> implements AdminKeys {
```

Comment:
since this is our internal impl class, not user-facing, Javadocs are not needed on `@override` members